### ISSUE TITLE
Redirect Request: Vet Centers/outstations still needing redirects

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#53460]
### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders as appropriate.  

### Type of request
- [ ] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)

### Implementation date
When does this request need to be live?
Choose one:
- [ ] On a specific date, connected to a program or publishing requirement. **DATE:**
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [ ] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL
---  |  ---
www.va.gov/find-locations/facility/vc_4061OS | https://www.va.gov/fargo-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=6443&dnum=All | https://www.va.gov/fargo-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=6443 | https://www.va.gov/fargo-vet-center/
www.va.gov/find-locations/facility/vc_0844MVC | https://www.va.gov/knoxville-vet-center/
www.va.gov/find-locations/facility/vc_0720V | https://www.va.gov/knoxville-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=6187&dnum=All | https://www.va.gov/knoxville-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=6187 | https://www.va.gov/knoxville-vet-center/
www.va.gov/find-locations/facility/vc_0806MVC | https://www.va.gov/lexington-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=652&dnum=All | https://www.va.gov/savannah-vet-center/
www.va.gov/find-locations/facility/vc_4421OS | https://www.va.gov/la-crosse-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=6177&dnum=All | https://www.va.gov/la-crosse-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=6177 | https://www.va.gov/la-crosse-vet-center/

### Vanity URLs 
Vanity URL  |  Landing page
---  |  ---
requested URL | landing page URL

**Link to campaign landing page request issue:**

### Process, Roles and Responsibilities
- [ ] Requesting team: Above information is provided
- [ ] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [ ] IA: Request is vetted and documented and implementation plan is clear 
- [ ] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by RLHecht** [comment]
These are Vet Centers and Outstations I noticed on our redirect spreadsheet that were never redirected when the Vet Center went live. There are a few more coming that I'm waiting for confirmation on and can add them to this ticket hopefully tomorrow. Or I can open a new request if that's easier. @jilladams I'll unassign you until these are approved.

**Comment by RLHecht** [comment]
@jennymayoco You can start on this since we're still waiting to hear back from the other ones I wanted to add on here. I can create a separate ticket for those once we get them or I'll let you know that I've added them to this one. Thanks!

**Comment by RLHecht** [comment]
@jennymayoco I just added the redirects I was waiting on (two outstations that redirect to San Juan). Thanks!

**Comment by jennymayoco** [comment]
@RLHecht I came across a couple of current links that come up blank:
https://www.va.gov/find-locations/facility/vc_3121OS
https://www.va.gov/find-locations/facility/vc_3122OS

They both redirect to the San Juan vet center page. Would you be able to confirm if the links above are correct?

**Comment by RLHecht** [comment]
@jennymayoco The first is for the St. Croix outstation and the second is for the St. Thomas outstation. I'm not sure why these are coming back as blank but I believe they're correct and should still have a redirect.

**Comment by jennymayoco** [comment]
hi @jilladams! These are good to move forward with
